cluster-scope/base/user.openshift.io/groups/odh-admin/group.yaml

```diff
@@ -3,17 +3,18 @@ kind: Group
 metadata:
   name: odh-admin
 users:
-  - anishasthana
+  - 4n4nd
   - accorvin
-  - maulikjs
-  - vpavlin
+  - anishasthana
+  - codificat
+  - crobby
+  - hemajv
+  - humairak
+  - HumairAK
   - lavlas
   - LaVLaS
-  - crobby
+  - maulikjs
   - nakfour
   - rimolive
-  - humairak
-  - HumairAK
-  - 4n4nd
   - tumido
-  - hemajv
+  - vpavlin
```

Comment:
@codificat can I ninja your edit and request you remove the following: 

- vpavlin
- maulikjs
- nakfour
- 4n4nd

Reply:
if you don't mind - feel free to add as a separate commit

Reply:
Sure! done (new commit added)